Sources/BranchSDK/Branch.m

```diff
@@ -1971,8 +1981,9 @@ - (void)initUserSessionAndCallCallback:(BOOL)callCallback sceneIdentifier:(NSStr
     
     dispatch_async(self.isolationQueue, ^(){
 
+        
         // If the session is not yet initialized
```

Comment:
Can you add a comment on this line that reset means we've got a new link we need to process

Reply:
Or whatever `reset` specifically indicates